controllers/release/release_adapter.go

```diff
@@ -185,14 +185,19 @@ func (a *Adapter) EnsureTargetContextIsSet() (results.OperationResult, error) {
 // will be extracted from the given ReleaseStrategy. The Release's ApplicationSnapshot will also be passed to the
 // release PipelineRun.
 func (a *Adapter) createReleasePipelineRun(releaseStrategy *v1alpha1.ReleaseStrategy, applicationSnapshot *appstudioshared.ApplicationSnapshot) (*v1beta1.PipelineRun, error) {
+	releasePlan, err := a.getReleasePlan()
```

Comment:
We already have the application name in the snapshot
https://github.com/redhat-appstudio/managed-gitops/blob/main/appstudio-shared/apis/appstudio.redhat.com/v1alpha1/applicationsnapshot_types.go#L31
Why not pull it from there instead of having to retrieve the release plan?

Reply:
I don't have an actual reason for have used the one contained in the Release Plan. When I saw it, it looked  to me more appropriate than the one in the snapshot. In your opinion, why the snapshot would be better (besides it is already passed as a parameter)?

Reply:
That is my only reason - so we don't have to pass more parameters to get information already present in other parameters

Reply:
I agree with Johnny. We can already get the application name from the snapshot, so no need to fetch the ReleasePlan.

Reply:
changed to get it from the snapshot.